desktop/core/src/desktop/api2.py

```diff
@@ -813,6 +814,44 @@ def export_documents(request):
     return make_response(f.getvalue(), 'json', filename)
 
 
+def topologicalSort(docs):
```

Comment:
Aren't we using snake_case rather than camelCase?

Reply:
Yes, got it Thanks